app/models/manageiq/providers/azure/cloud_manager/event_catcher/runner.rb

```diff
@@ -49,6 +45,8 @@ def reset_event_monitor_handle
 
   def filtered?(event)
     event_type = parse_event_type(event)
-    filtered_events.include?(event_type)
+    filtered_events.include?(event_type).tap do |result|
+      _log.info "#{log_prefix} Skipping filtered Azure event #{event_type} for #{event["resourceId"]}" if result
+    end
```

Comment:
> if you hate tap 

:) I actually like tap, but for:

> Yields self to the block, and then returns self. The primary purpose of this method is to “tap into” a method chain, in order to perform operations on intermediate results within the chain.

So, I find it a bit strange to tap into a boolean result and perform something on it. But as I read it more throughout this PR I got accustomed to it :) And it's actually a neat and compact way to use in this case. So thanks @bzwei for showing me this.

Do you both happen to know if this is common style in other projects?